packages/core/src/Typography/Typography.tsx

```diff
@@ -92,14 +92,15 @@ export const HvTypography: <C extends React.ElementType = "p">(
       classes: classesProp,
       variant: variantProp = "body",
       link = false,
-      disabled = false,
       noWrap = false,
       paragraph = false,
       ...others
     } = useDefaultProps("HvTypography", props);
     const { classes, cx } = useClasses(classesProp);
     const { activeTheme } = useTheme();
 
+    const { disabled } = props;
```

Comment:
If we want to forward this to the component, we should forward it directly (ie. `disabled={disabled}`) - destructuring in multiple places makes it harder to follow what is/isn't forwarded

Reply:
I did that but the same from above was happening. But I'll try again to see if I missing something.

Reply:
Updated ✅